samples/highcharts/demo/neural-network/demo.js

```diff
@@ -0,0 +1,101 @@
+const layers = [1, 6, 6, 6, 2];
```

Comment:
Add a comment that those variables can be modified to render different graphs.
Unless we want to go with a fixed dataset, leaving the decision up to @TorsteinHonsi

Reply:
+1 for the comment. I'd like to see some more comments in general for this demo.